libmscore/album.cpp

```diff
@@ -0,0 +1,276 @@
+                reader.readElementText();
+            }
+        } else if (tag == "enabled") {
+            enabled = reader.readBool();
+        } else {
+            reader.skipCurrentElement();
+        }
+    }
+}
+
+//---------------------------------------------------------
+//   saveAlbumItem
+//---------------------------------------------------------
+
+
+void AlbumItem::writeAlbumItem(XmlWriter &writer)
+{
+    writer.stag("Score");
+    writer.tag("alias", "");
+    writer.tag("path", _fileInfo.absoluteFilePath());
+    QDir dir(album->_fileInfo.dir());
+    QString relativePath = dir.relativeFilePath(_fileInfo.absoluteFilePath());
+    writer.tag("relativePath", relativePath);
+    writer.tag("enabled", enabled);
+    writer.etag();
+}
+
+//---------------------------------------------------------
+//---------------------------------------------------------
+//
+//   Album
+//
+//---------------------------------------------------------
+//---------------------------------------------------------
+
+Album* Album::activeAlbum = nullptr;
+
+Album::Album()
+{
+
+}
+
+//---------------------------------------------------------
+//   addScore
+//---------------------------------------------------------
+
+void Album::addScore(MasterScore *score, bool enabled)
+{
+    if (!score) {
+        std::cout << "There is no score to add to album..." << std::endl;
+        return;
+    }
+    std::cout << "Adding score to album..." << std::endl;
+
+    AlbumItem* albumScore = new AlbumItem(this, score, enabled);
+    _albumItems.push_back(albumScore);
+}
+
+//---------------------------------------------------------
+//   removeScore
+//---------------------------------------------------------
+
+MasterScore* Album::removeScore(MasterScore *score)
+{
+    return nullptr;
+}
+
+MasterScore* Album::removeScore(int index)
+{
+    _albumItems.erase(_albumItems.begin() + index);
+    return nullptr;
+}
+
+//---------------------------------------------------------
+//   swap
+//---------------------------------------------------------
+
+void Album::swap(int indexA, int indexB)
+{
+    std::swap(_albumItems.at(indexA), _albumItems.at(indexB));
+}
+
+//---------------------------------------------------------
+//   scoreInAlbum
+//---------------------------------------------------------
+
+bool Album::scoreInActiveAlbum(MasterScore *score)
+{
+    if (!activeAlbum)
+        return false;
+
+    for (auto x : activeAlbum->_albumItems) {
+        if (x->score == score) {
+            return true;
+        }
+    }
+    return false;
+}
+
+//---------------------------------------------------------
+//   getDominant
+//---------------------------------------------------------
+
+MasterScore *Album::getDominant()
+{
+    return _albumItems.front()->score;
+}
+
+//---------------------------------------------------------
+//   loadFromFile
+//---------------------------------------------------------
+
+bool Ms::Album::loadFromFile(const QString &path)
```

Comment:
I would think so too, those were from before I found out how to edit my QtCreator settings :-)

Reply:
And where did you find that out?

Reply:
@Marr11317 QtCreator Menubar -> Tools -> Options -> C++, then:

1. Select the built-in Qt style.
2. Press `Copy...` (because you can't edit the built-in style.
3. Press `Edit...`
4. Go to `Pointers and References`.
5. Deselect `Identifier` and select `Type name` (and maybe left const/volatile?).